site/app/templates/admin/LateDays.twig

```diff
@@ -18,8 +18,8 @@
                 <input class="option-input" type="text" name="datestamp" id="datestamp">
```

Comment:
I would suggest changing "Datestamp" to something more clear if an instructor doesn't read the line at the top, something "Time when to grant/activate latedays"

Reply:
I agree. Does it look more clear to you now?